geo/src/algorithm/bool_ops/assembly.rs

```diff
@@ -85,14 +98,15 @@ impl<T: GeoFloat> RegionAssembly<T> {
                         parent_snake_idx = Some(
                             iter.prev_active(d)
                                 .map(|(_, seg)| seg.snake_idx.get())
+                                // TODO
```

Comment:
Can you remove these TODOs?

Reply:
I can, but these TODOs really need an explanation. They make no sense to me. I tried to remove these defaults as well and tests started to fail, so clearly they are necessary but I don't have a logical reason *how*.

Reply:
@rmanoka Do you recall why this `.unwrap_or(0)` is here?

Reply:
I don't think the number here matters.  If the expression is not `Some()`, then the "snake" will complete to an exterior ring, which doesn't need a parent idx.  I tried verifying by setting random numbers inside `.unwrap_or()`, and tests seem to pass.

Reply:
@andriyDev Are you OK with this explanation?

Reply:
I've added the explanation to the comments. I believe this is ready to go then (unless there are other concerns). :)